scripts/coccinelle/identifier_length.cocci

```
@@ -3,6 +3,20 @@
 
 virtual report
 
+@initialize:python
+  depends on report
+@
+@@
+identifiers = {}
+
+def check_identifier(id, pos):
```

What is `pos` here?

---

we apply it only in the diff, we don't want to report an error or block a pr for something that was changed

---

I do no understand. Put it in a more context.

---

The way coccinelle works is in the whole file, but I don't want to report all the errors in the file, but only in the code that was changed.